src/universal/index.ts

```diff
@@ -19,6 +19,48 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/**
+ * :::{admonition} Experimental API
+ * :class: important
+ *
+ * This API is still experimental, and subject to change. It builds on top of
+ * both ACP and ACL, aiming at being adaptable to any Access Control system that
+ * may be implemented in Solid. That is why it is purely Resource-centric: the
+ * library discovers metadata associated with the Resource itself, and calls the
+ * appropriate underlying API to deal with the Access Control in place for the
+ * target Resource.
```

Comment:
```suggestion
 * This API is still experimental, and subject to change. It builds on top of
 * both ACP and WAC, aiming at being adaptable to any Access Control system that
 * may be implemented in Solid. That is why it is purely Resource-centric: the
 * library discovers metadata associated with the Resource itself, and calls the
 * appropriate underlying API to deal with the Access Control in place for the
 * target Resource.
```

Reply:
Heads up: this comes from https://github.com/inrupt/solid-client-js/blob/4badfc72b390e35e06454c6e550bc67c79f8c491/src/index.ts#L371-L389